### PR TITLE
Thomas rochais/fix jumping menu

### PR DESCRIPTION
**Pull request by ThomasRochais**
### What changed:
1. The menu cursor no longer jumps when coming back to the menu
2. The rocket can move diagonally
3. The rocket's orientation has been changed to horizontal
4. The rocket's initial position has been moved to the left

### CODE REVIEW

**Review by Hamtara — COMMENTED** [review]
Seems to works fine on my side 👍

File: rocket.py [hunk 1]
```diff
@@ -7,29 +7,46 @@ def __init__(self, game):
         self.game = game  # Use a game object
         # Initial position of the rectangle (bottom, centered)
         self.spaceship = pygame.image.load(os.path.join("assets/sprites", "spaceship-a.svg"))
+        # Rotate the spaceship 90degrees clockwise
+        self.spaceship = pygame.transform.rotate(self.spaceship,-90)
         self.width, self.height = self.spaceship.get_size()
         # Rescale spaceship
```
[the remark this answers is not in the document]
Comment (ThomasRochais) [reply rocket.py:13]: Fair enough.

File: game.py [hunk 2]
```diff
@@ -59,6 +52,16 @@ def check_events(self):
                 if event.key == pygame.K_RIGHT:
                     self.RIGHT_KEY = True
 
+    def move_rocket(self):
```
[the remark this answers is not in the document]
Comment (ThomasRochais) [reply game.py:68]: good point. I got rid of the individual moving functions from the Rocket class and put them all in the same move move_rocket function

File: rocket.py [hunk 3]
```diff
@@ -17,19 +17,19 @@ def __init__(self, game):
         self.init_pos('LEFT')
```
[the remark this answers is not in the document]
Comment (ThomasRochais) [reply rocket.py:17]: Oh! I didn't know about property(), thanks!
I'm gonna give it a try.

[the remark this answers is not in the document]
Comment (ThomasRochais) [reply rocket.py]: Actually, don't see the use of a property there. I just renamed it and I now use it in game.py to reset the position of the rocket when exiting the game and coming back to it.

**Review by Hamtara — APPROVED** [review]

**Pull request merged**